utils/gyb_syntax_support/protocolsMap.py

```diff
@@ -0,0 +1,20 @@
+PROTOCOL_CONFORMANCES_MAP = {
```

Comment:
Not sure if this names is a good fitting as it contains structs and protocols with their conformances